### PR TITLE
fix(drawer): scroll position for longer content

### PR DESCRIPTION
**Pull request by maulik-thaker1**
## Summary

- fixed side drawer body scroll when body content inside is too long

## Screenshots

Resolved below issue :

Screenshot 2024-03-04 at 10 34 12 AM

### CODE REVIEW

**Review by brian-patrick-3 — CHANGES_REQUESTED** [review]
I think there is a simpler fix here.  Just remove `height: 100vh` from `.drawer-content-wrapper`.

**Comment by maulik-thaker1** [comment]
> height: 100vh

Nope I did that the dialog body cut behind the footer. hence I did this.

**Comment by maulik-thaker1** [comment]
> > height: 100vh
> 
> Nope I did that the dialog body cut behind the footer. hence I did this.

Screenshot 2024-03-04 at 8 23 37 PM


its look like this.

**Comment by brian-patrick-3** [comment]
You can probably fix that with `display: flex;` on the dialog. I'll give an example.

**Comment by maulik-thaker1** [comment]
> You can probably fix that with `display: flex;` on the dialog. I'll give an example.

Did same. I was not comfortable for negative (ex: -32px) margin so avoided that time. But it's look similar so I applied flex.

File: src/components/reusable/sideDrawer/sideDrawer.scss [hunk 1]
```diff
@@ -38,6 +41,13 @@ dialog {
   padding: 0 32px 32px 32px;
   height: 100vh;
   margin-bottom: 10px;
+  overflow-y: scroll;
+  &-height {
+    height: calc(100vh - 105px);
+    @media (max-width: 42rem) {
+      height: calc(100vh - 163px);
+    }
+  }
```
[the remark this answers is not in the document]
Comment (maulik-thaker1) [reply src/components/reusable/sideDrawer/sideDrawer.scss]: Done.

**Review by brian-patrick-3 — APPROVED** [review]

**Pull request merged**